Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -391,20 +401,24 @@ def fetch_events_command(
         fetch_limit: limit of events in a fetch
         config_ids: security configuration ids to fetch, e.g. `51000;56080`
         ctx: The integration context
+        page_size: The number of events to limit for every request.
 
     Yields:
         (list[dict], str, int, str): events, new offset, total number of events fetched, and new last_run time to set.
     """
     total_events_count = 0
     from_epoch, _ = parse_date_range(date_range=fetch_time, date_format='%s')
     offset = ctx.get("offset")
+    hashed_events_from_previous_run = ctx.get("hashed_events_from_previous_run", [])
     while total_events_count < int(fetch_limit):
-        demisto.info(f"Preparing to get events with {offset=}, {from_epoch=}, and {fetch_limit=}")
-        events, offset = client.get_events_with_offset(config_ids, offset, FETCH_EVENTS_PAGE_SIZE, from_epoch)
+        demisto.info(f"Preparing to get events with {offset=}, {page_size=}, and {fetch_limit=}")
+        events, offset = client.get_events_with_offset(config_ids, offset, page_size, from_epoch)
         if not events:
             demisto.info("Didn't receive any events, breaking.")
             break
+        hashed_events_from_current_run = []
         for event in events:
+            hashed_events_from_current_run.append(json.dumps(event, sort_keys=True))
```

Comment:
these are json, not hashed. lets use a hashing algorithm - the data doesnt actually get used